prep_sim_rirs.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# Script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000     # Sampling rate of the output RIR waveform
+output_bit=16          # bits of each sample in the RIR waveform
+num_room=50            # number of rooms to be sampled
+rir_per_room=100       # number of RIR to be sampled for each room
+prefix="large-"                # prefix to the RIR id
+room_lower_bound=10    # lower boung of the room length and width
+room_upper_bound=30    # upper boung of the room length and width
+rir_duration=2         # duration of the output RIR waveform in secs
+generator="./rir_generator.cpp"   # path to the RIR generator
+
+
+if [ $# != 1 ]; then
+  echo "Usage: "
+  echo "  $0 [options] <output-dir>"
+  echo "e.g.:"
+  echo " $0  data/simulated_rirs"
+  exit 1;
+fi
+
+output_dir=$1
+mkdir -p $output_dir
+
+cat >./genrir.m <<EOF

```

Comment:
I suggest to add  a separate file, as it is easy to read code with syntax highlighting.